packages/server/api/webscrape.py

```diff
@@ -3,32 +3,63 @@
 import re
 from urllib import parse
 from bs4 import BeautifulSoup
+import json
 
 class handler(BaseHTTPRequestHandler):
   whitelisted_url_regexes = [
     re.compile('^https:\/\/www\.bonappetit\.com')
   ]
 
+  def error_handler(error_string, error_code = 500):
```

Comment:
Does this handle all errors, including uncaught exceptions? Or just when we call it ourselves?

Reply:
Just when we call it ourselves

Reply:
so this still doesn't solve for uncaught exceptions, meaning we still are going to have problems :/

Reply:
I think we should make another ticket to add a python error handling framework